specification/attestation/data-plane/Microsoft.Attestation/stable/2018-09-01-preview/attestation.json

```diff
@@ -514,7 +514,16 @@
       "type": "object",
       "properties": {
         "policy": {
-          "description": "String-encoded attestation policy document.",
+          "description": "JSON Web Token whose body is an AttestationPolicyRequest definition.",
+          "type": "string"
+        }
+      }
+    },
+    "AttestationPolicyRequest": {
```

Comment:
>AttestationPolicyRequest [](start = 5, length = 24)

This doesn't appear to be used.

Reply:
Actually AttestationPolicy is used (this was a mistake that I'm fixing right now). What isn't use is AttestationPolicyRequest. However the body of the JWS specified as input to the Policy_PrepareToSet API is an AttestationPolicyRequest, so the type is used in the API even if it's not explicitly mentioned in the swagger.

Reply:
@#$@#$. I have to remove AttestationPolicyRequest from the public swagger and replace it with a text comment because the swagger won't validate if I use the actual struct definition. It's a shame, but the release preview API surface is fixed and adding this type would introduce a breaking change (because it broke the service when I tested the generated REST APIs :().